src/commands/lambda/functions/instrument.ts

```diff
@@ -333,6 +335,18 @@ export const calculateUpdateRequest = async (
     if (settings.interactive && !settings.extensionVersion) {
       extensionVersion = await findLatestLayerVersion(EXTENSION_LAYER_KEY as LayerKey, region)
     }
+
+    if (settings.fips) {
+      const currentSite = site ?? oldEnvVars[SITE_ENV_VAR] ?? 'datadoghq.com'
+      if (currentSite !== DATADOG_SITE_GOV) {
+        console.warn('DD_SITE is not set to ddog-gov.com, FIPS will not be fully compliant')
```

Comment:
> FIPS will not be fully compliant

I think we should rephrase this – it's not "FIPS" that would "not be fully compliant"

Reply:
Also, can you pass `context: CommandContext` to this function and replace this line with:
```suggestion
        context.stderr.write('FIPS mode was enabled on the extension layer, but DD_SITE is not set to ddog-gov.com\n')
```